packages/sns/candid/sns_index.did

```diff
@@ -47,7 +47,7 @@ type TransactionWithId = record {
 type GetTransactions = record {
   transactions : vec TransactionWithId;
   // The txid of the oldest transaction the account has
-  oldest_tx_id : opt nat64;
+  oldest_tx_id : opt TxId;
```

Comment:
In your PR #219 you had issue with `TxId` which if I reckon you had to manually overwrite. Is it safe to have here `TxId` again?

Reply:
Yes when using the new SNS Index canisters. I tested it against small11 and it was working. I saw a transactions and everything worked well.

The problem is that to work with transactions we can only work with small11 at the moment. Which is the only environment with the latest sns canisters.

Reply:
Ok

Reply:
Thanks for the feedback